### DISCUSSION TITLE
Responsive modal problem

### DISCUSSION THREAD

**Discussion by bptrsn** [saadeghi/daisyui#668] [Q&A] [answered]
I'm trying to use the responsive modal from https://daisyui.com/components/modal/ - the one at the bottom.
It's working on the https://daisyui.com/components/modal/ demo page but not when I copy the modal to my own html file.
I always get the modal on the bottom, unless I remove modal-bottom class.
I've tried sm/md/lg prefixes.

I'm loading daisyui/tailwind from CDN as on https://daisyui.com/docs/cdn/

Any suggestions what I'm doing wrong?

**Comment by saadeghi** [comment, marked as answer]
Looks like an issue with the CDN version.  
I created #675 issue for it. Thanks for reporting

**Reply by bptrsn** [reply]
Great, thanks for fixing.
